homework.md

```diff
@@ -92,13 +94,13 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Save your changes
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Visual Studio
```

Comment:
Hi Simba, Everything looks great only thing I noticed would be that at CodeYourFuture we use VS Code, which as its still part of Visual Studio its not as Powerful or as draining on your Computer